Packs/GroupIB_ThreatIntelligenceAttribution/Scripts/GIBIncidentUpdate/GIBIncidentUpdate.py

```diff
@@ -0,0 +1,23 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+result = True
+inc = demisto.incident()
+custom_fields = inc.get("CustomFields", {})
+del inc["CustomFields"]
+del inc["labels"]
+del inc["occurred"]
+del inc["sla"]
+inc.update(custom_fields)
+demisto.error(custom_fields)
```

Comment:
why is this as `demisto.error`? what is the purpose? if this is not an actual error (as it looks like) perhaps there is a better command for that purpose.

Reply:
done